spec/helpers/sign_in.rb

```diff
@@ -0,0 +1,13 @@
+module SignIn
+  def sign_in(user)
+    visit root_path
+    click_link 'Sign In'
+    fill_in 'Email', with: user.email
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.